apps/pie-docs/src/assets/styles/components/_contentList.scss

```diff
@@ -13,14 +13,14 @@ $tooltip-border-radius: 100px;
     max-width: $contentList-maxWidth;
 
     ol {
-        column-width: $contentList-column-width 1;
         column-gap: var(--dt-spacing-f);
         padding: 0;
         column-fill: auto;
+        column-count: 1;
 
         @include f.media('>=mid') {
             max-height: $contentList-maxHeight;
-            column-count: var(--column-count);
```

Comment:
This was being set by the [plugin](https://www.npmjs.com/package/eleventy-plugin-toc) before. It doesn't seem necessary as if we just set to 3 columns, it will only take up 2 if the content is not large enough and will cap at 3.